### PR TITLE
Action step documentation

### PR DESCRIPTION
**Pull request by shubham149**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by jimsheldon — APPROVED** [review]

**Review by ravilach — APPROVED** [review]
Looks good on my end. @KimberlyFields mind touting this for a CI review?

**Comment by ravilach** [comment]
@shubham149 can you add one more card to the AllTutorials.tsx also please under the CI section?

**Review by douglas-j-bothwell — APPROVED** [review]
I implemented my edits in the last commit. Approved.

File: src/components/LandingPage/AllTutorials.tsx [hunk 1]
```diff
@@ -189,6 +189,20 @@ const CIList: CardItem[] = [
     time: "15 min",
     link: "/tutorials/build-code/ci-tutorial-container-signing",
   },
+  {
+    title: "Github Action steps",
```
Comment (rustd) [inline src/components/LandingPage/AllTutorials.tsx:193]: Use Github action step

File: tutorials/build-code/ci-github-action-step.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
+# Github action step
```
Comment (rustd) [inline tutorials/build-code/ci-github-action-step.md:1]: Use Github action in Harness CI

**Pull request merged**